Controllers/LaunchController.cs

```diff
@@ -8,36 +11,175 @@ namespace SpaceLaunchAPI.Controllers
     [ApiController]
     public class LaunchController : ControllerBase
     {
+        private readonly ILaunchRepository launchRepo;
+
+        public LaunchController(ILaunchRepository launchRepo)
+        {
+            this.launchRepo = launchRepo;
+        }
         // GET: api/<LaaunchController>
         [HttpGet]
-        public IEnumerable<string> Get()
+        public async Task<IActionResult> Get()
         {
-            return new string[] { "value1", "value2" };
+            var LaunchesDomain = await launchRepo.GetAllAsync();
+            var LaunchesDTO = LaunchesDomain.Select(x => LaunchDomainToDTO(x));
+            return Ok(LaunchesDTO);
         }
 
         // GET api/<LaaunchController>/5
         [HttpGet("{id}")]
-        public string Get(int id)
+        public async Task<IActionResult> Get(string id)
```

Comment:
Where a model, in our case DTO is returned, lets keep type safety by ensuring the return type is included.
Task<ActionResult<*****DTO>> or Task<ActionResult<IEnumerable<***DTO>>> for lists